install.sh

```diff
@@ -1,32 +1,65 @@
 #!/bin/bash
+# Installs bash and python dependencies
+# OVERWRITES /etc/ config files
 
-# Install Dependencies
-sudo apt-get update
-sudo apt-get upgrade
-sudo apt-get install hostapd
-sudo apt-get install dnsmasq
+# Colors
+ERROR=$(tput setaf 1)
+SUCESS=$(tput setaf 2)
+MESSAGE=$(tput setaf 6)
+NORMAL=$(tput sgr0)
 
-# Disable Services
-sudo systemctl disable hostapd
-sudo systemctl disable dnsmasq
+# User warning
+printf "\n\n${ERROR}WARNING:${NORMAL} This script will overwrite your system configuration files:\n"
+printf "\n\t/etc/hostapd/hostapd.conf"
+printf "\n\t/etc/default/hostapd"
+printf "\n\t/etc/dnsmasq.conf"
+printf "\n\t/etc/dhcpcd.conf"
+printf "\n\t/usr/bin/hotspotWebserver.py"
```

Comment:
Line 17-20 are my scripts. They aren't being overwritten. We should leave the other warnings though